modules/installation-and-upgrade/pages/container-deployment/suma/sle-micro-deployment.adoc

```diff
@@ -199,30 +244,30 @@ INF Setting up uyuni network
 
 . Wait for deployment to complete.
 
-. Open a browser and proceed to your servers FQDN.
+. Open a browser and proceed to your server's FQDN or IP address.
 
-In this section you learned how to deploy an {productname} Server container.
+In this section you learned how to deploy a {productname} Server container.
```

Comment:
```suggestion
In this section you learned how to deploy the {productname} Server container.
```

if {productname} is SUSE Manager, is it correct to  write `a` {productname}, but if this is Uyuni, it should  be `an` {productname}. What about writing `the` {productname} instead?